autosupport/README.md

```diff
@@ -24,2314 +24,4229 @@ An example payload sent by Harvest looks like this. You can see exactly what Har
-          "aggr-attributes aggr-inode-attributes files-total",
-          "aggr-attributes aggr-inode-attributes files-used",
-          "aggr-attributes aggr-inode-attributes inodefile-private-capacity",
-          "aggr-attributes aggr-inode-attributes inodefile-public-capacity",
-          "aggr-attributes aggr-inode-attributes maxfiles-available",
-          "aggr-attributes aggr-inode-attributes maxfiles-possible",
-          "aggr-attributes aggr-inode-attributes maxfiles-used",
-          "aggr-attributes aggr-inode-attributes percent-inode-used-capacity",
-          "aggr-attributes aggr-space-attributes capacity-tier-used",
-          "aggr-attributes aggr-space-attributes data-compacted-count",
-          "aggr-attributes aggr-space-attributes data-compaction-space-saved",
-          "aggr-attributes aggr-space-attributes data-compaction-space-saved-percent",
-          "aggr-attributes aggr-space-attributes hybrid-cache-size-total",
-          "aggr-attributes aggr-space-attributes percent-used-capacity",
-          "aggr-attributes aggr-space-attributes performance-tier-inactive-user-data",
-          "aggr-attributes aggr-space-attributes performance-tier-inactive-user-data-percent",
-          "aggr-attributes aggr-space-attributes physical-used",
-          "aggr-attributes aggr-space-attributes physical-used-percent",
-          "aggr-attributes aggr-space-attributes sis-shared-count",
-          "aggr-attributes aggr-space-attributes sis-space-saved",
-          "aggr-attributes aggr-space-attributes sis-space-saved-percent",
-          "aggr-attributes aggr-space-attributes size-available",
-          "aggr-attributes aggr-space-attributes size-total",
-          "aggr-attributes aggr-space-attributes size-used",
-          "aggr-attributes aggr-space-attributes total-reserved-space",
-          "aggr-attributes aggr-volume-count-attributes flexvol-count",
-          "aggr-attributes aggr-snapshot-attributes files-total",
-          "aggr-attributes aggr-snapshot-attributes files-used",
-          "aggr-attributes aggr-snapshot-attributes maxfiles-available",
-          "aggr-attributes aggr-snapshot-attributes maxfiles-possible",
-          "aggr-attributes aggr-snapshot-attributes maxfiles-used",
-          "aggr-attributes aggr-snapshot-attributes percent-inode-used-capacity",
-          "aggr-attributes aggr-snapshot-attributes percent-used-capacity",
-          "aggr-attributes aggr-snapshot-attributes size-available",
-          "aggr-attributes aggr-snapshot-attributes size-total",
-          "aggr-attributes aggr-snapshot-attributes size-used",
-          "aggr-attributes aggr-snapshot-attributes snapshot-reserve-percent"
-        ]
-      }
-    },
-    {
-      "Name": "Zapi",
-      "Query": "volume-get-iter",
-      "BatchSize": "500",
-      "ClientTimeout": "180",
-      "Schedules": [
-        {
-          "Name": "instance",
-          "Schedule": "600s"
-        },
-        {
```

Comment:
These zapi payloads are removed from here, as I couldn't find them in right side changes ?